packages/wdio-cucumber-framework/src/index.ts

```diff
@@ -124,8 +124,17 @@ class CucumberAdapter {
 
         /**
          * formatting options used by custom cucumberFormatter
+         * https://github.com/cucumber/cucumber-js/blob/main/docs/formatters.md#options
```

Comment:
I'm personally not a fan of putting links in comments as they become outdated.

Reply:
Let's replace `main` with the actual sha value so the link stays relevant.